python/podio_class_generator.py

```diff
@@ -489,7 +490,8 @@ def create_class(self, classname, definition):
         if "includes" in extra:
             extraIncludes = set(extra["includes"].split('\n'))
             datatype["includes"].update(extraIncludes)
-            print " ***** adding includes : " ,  extraIncludes , "to" ,  datatype["includes"]
+            if self.verbose:
```

Comment:
Should this just be dropped?

Reply:
I think this was just debugging during development. It should be dropped indeed